src/main/java/com/feywild/feywild/world/gen/OreType.java

```diff
@@ -123,4 +107,23 @@ public PlacedFeature getAlfheimFeature() {
     public HeightProvider getHeight() {
         return UniformHeight.of(VerticalAnchor.absolute(getMinHeight()), VerticalAnchor.absolute(getMaxHeight()));
     }
+
+    public LazyValue<PlacedFeature> setFeature(Block block, RuleTest oreRule, Boolean isRare) {
+
+        BlockState defaultBlock = block.defaultBlockState();
+
+        List<PlacementModifier> modifier = isRare ?
+                OrePlacement.rareOrePlacement(getSpawnWeight(),
+                        HeightRangePlacement.triangle(VerticalAnchor.absolute(getMinHeight()), VerticalAnchor.absolute(getMaxHeight()))) :
+                OrePlacement.commonOrePlacement(getSpawnWeight(),
```

Comment:
The `commonOrePlacement` should be passed as a method reference. I can change that.